From f5966b72599ddc1335403cff13159e872dd95007 Mon Sep 17 00:00:00 2001
Message-Id: <f5966b72599ddc1335403cff13159e872dd95007@dist-git>
From: John Ferlan <jferlan@redhat.com>
Date: Tue, 18 Aug 2015 14:47:13 -0400
Subject: [PATCH] api: Remove check on iothread_id arg in virDomainPinIOThread

https://bugzilla.redhat.com/show_bug.cgi?id=1251886

Allow 0 as an iothread_id and force the hypervisor to handle.
The qemuDomainPinIOThread API will look up the iothread_id of
0 and not find it and message that anyway.

(cherry picked from commit d64b81a8f0b0a4660691565cde65b13df2647391)
Signed-off-by: John Ferlan <jferlan@redhat.com>
Signed-off-by: Jiri Denemark <jdenemar@redhat.com>
---
 src/libvirt-domain.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/libvirt-domain.c b/src/libvirt-domain.c
index 837933f..c4377bf 100644
--- a/src/libvirt-domain.c
+++ b/src/libvirt-domain.c
@@ -7926,7 +7926,6 @@ virDomainPinIOThread(virDomainPtr domain,
     conn = domain->conn;
 
     virCheckReadOnlyGoto(conn->flags, error);
-    virCheckPositiveArgGoto(iothread_id, error);
     virCheckNonNullArgGoto(cpumap, error);
     virCheckPositiveArgGoto(maplen, error);
 
-- 
2.5.0

